### ISSUE TITLE
Downloads of small files initially confuse the user with the pop-up menu

### ISSUE THREAD

**Issue by erangell** [beakerbrowser/beaker#891]
I downloaded a small file several times by repeatedly clicking the download button before I realized that the pop-up menu was intended to allow you to browse your downloads.  When I downloaded a larger file I saw a progress bar and it made sense.  I'm used to seeing the Chrome download icon at the bottom of the screen, then clicking a button to view my downloads.  This is just a nice-to-have feature for a future release.
Operation System: Mac OSX Sierra
Beaker version: latest

**Comment by pfrazee** [comment]
Yeah I can confirm, that is easy to miss. We'll look into something better. Thanks!
